api/app/tests/morecast_v2/test_morecast_v2_endpoint.py

```diff
@@ -72,11 +59,13 @@ def test_post_forecast_authorized(client: TestClient,
     """ Allowed to post station changes with correct role"""
 
     def mock_admin_role_function(*_, **__):
-        return MockJWTDecodeWithRole('forecaster')
+        return MockJWTDecodeWithRole('morecast2_write_forecast')
 
     monkeypatch.setattr(decode_fn, mock_admin_role_function)
 
-    response = client.post(morecast_v2_post_url, json=[forecast.dict()])
+    print(forecast)
```

Comment:
Stray `print`

Reply:
Thanks! Removed in https://github.com/bcgov/wps/pull/2725/commits/68230b9603c6f444b44e62b0a3bcd283cfbba0a2